.lighthouserc.js

```diff
@@ -22,26 +22,31 @@ module.exports = {
       'numberOfRuns': 3,
       'puppeteerScript': 'puppeteer-login-script.js',
       'url': [
-        'http://127.0.0.1:8181/signup?return_url=%2F',
-        'http://127.0.0.1:8181/',
         'http://127.0.0.1:8181/admin',
+        `http://127.0.0.1:8181/create/${process.env.exploration_editor}`,
+        `http://127.0.0.1:8181/collection_editor/create/${process.env.collection_editor}`,
+        `http://127.0.0.1:8181/topic_editor/${process.env.topic_editor}`,
+        `http://127.0.0.1:8181/skill_editor/${process.env.skill_editor}`,
```

Comment:
For each added path into this section, add the corresponding section below
```javascript
{
  'matchingUrlPattern': 'http://[^/]+/$',
  'assertions': {
    'uses-webp-images': [
      'error', {'maxLength': 0, 'strategy': 'pessimistic'}
    ]
  }
},
```

Reply:
done